### ISSUE TITLE
tmt test import failing on some makefiles

### ISSUE THREAD

**Issue by sbertramrh** [teemtee/tmt#1127]
If the makefile does not follow below structure for run entry it fails. 
run: build
	./runtest.sh

Attached are examples and hopefully all occurrences of this in /kernel repo.
[kernel.txt](https://github.com/teemtee/tmt/files/8389671/kernel.txt)

List was created using the following:

find . -name Makefile -exec grep -A 2 "run: build" {} \; -print > ~/kernel.txt

So every fourth line contains the location of the Makefile relative to /kernel.

**Comment by lukaszachy** [comment]
Could you try with latest copr build? If it was about missing 'build' target it will be fixed.
If not, we have more bugs to fix in convert.

**Comment by juk** [comment]
Some Makefiles fail to import due to a "run" target is more then two lines, ex
```
run: build
	@echo "============ Run Tests ============"
	LANG=C ./runtest.sh
```
The matching regex expects two lines:
`        regex_test = r'^run:.*\n\t(.*)$'`

**Comment by thrix** [comment]
@lukaszachy so is the expectation here that the automatic imported adds both `build` and `run` target scripts?

I would think in cases like this manual import is needed, not complicate our lives

**Comment by lukaszachy** [comment]
@thrix Expectation is that if Makefile deviates from the simplest beaker-wizard produced Makefile (e.g. multiple lines in run target) then tmt warns user but leaves it up to them.

The linked PR just make import work when someone deleted 'build' target. At the second look it is a unused code (the only usage is to produce the warning when multiple lines are present in the target).

**Comment by psss** [comment]
> Expectation is that if Makefile deviates from the simplest beaker-wizard produced Makefile (e.g. multiple lines in run target) then tmt warns user but leaves it up to them.

Yes, we were not able to cover all special cases. Some custom `Makefile` tweaks just have to be handled manually.
